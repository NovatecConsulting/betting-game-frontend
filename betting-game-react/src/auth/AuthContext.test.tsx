import React from "react";
import { shallow } from "enzyme";
import { AuthContext } from ".";

describe(AuthContext.name + " component", () => {
    it("should render children and match snapshot.", () => {
        const wrapper = shallow(
            <AuthContext>
                <div>Test</div>
            </AuthContext>
        );

        expect(wrapper.find("div")).toHaveLength(1);
        expect(wrapper).toIncludeText("Test");
        expect(wrapper).toMatchSnapshot();
    });
});
