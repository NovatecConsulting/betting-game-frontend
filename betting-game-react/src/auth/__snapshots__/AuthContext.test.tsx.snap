// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthContext component should render children and match snapshot. 1`] = `
<Fragment>
  <div>
    Test
  </div>
</Fragment>
`;
